src/components/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useContext } from 'react';
+import { Input, Menu, Radio } from 'semantic-ui-react';
+import { Link, useHistory } from 'react-router-dom';
+import { useAuth } from '../../providers/Auth/Auth.provider';
+
+import VideosContext from '../../context/VideosContext';
+import ThemeContext from '../../context/ThemeContext';
+
+const MenuBar = () => {
+  const history = useHistory();
+  const { light, handleTheme } = useContext(ThemeContext);
+  const { handleHitEnter, handleOnChange } = useContext(VideosContext);
+
+  const { authenticated, logout } = useAuth();
+  const { pathname } = window.location;
+  const path = pathname === '/' ? 'home' : pathname.slice(1);
```

Comment:
What does it happen if the name of your path change in the future, you should change all the places where this 'home' appears right? So try to generate a single file for constants path names and import it wherever you must use it i.e. 
```
export const path = {
  HOME: 'home',
  VIDEOS: 'videos',
  ...
}
```